### DISCUSSION TITLE
FLET 2024 ROADMAP

### DISCUSSION THREAD

**Discussion by Reddy-Ganesh** [flet-dev/flet#2522] [Q&A]
### Question

1.In the Flet Development 2024 roadmap, I have observed the inclusion of new controls such as audio recorder, video, Google Ads, etc. Could you please provide an estimate for the availability of all these controls in Flet with Python?
 2. While utilizing those controls, it was indicated that there is no such control in Flet version 0.19.0.
![Screenshot 2024-02-02 121515](https://github.com/flet-dev/flet/assets/104451685/a148a6d7-ea3a-4b1e-be11-aa588d749b49)

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Controls marked with ✅ are those already present. On the other hand, those without are not yet available. (Click on the links to the controls you wish to have and vote+subscribe to the respective issues for updates)

[Here](https://github.com/orgs/flet-dev/projects/3/), you can have a global look at the work made by the core team.

**Reply by Reddy-Ganesh** [reply]
Okay Dude thanks for your information . Is there any idea about 2024 controls release date
